packages/frontend/index.d.ts

```diff
@@ -12,6 +12,7 @@ interface ArticleProps {
 type Pillar = 'news' | 'opinion' | 'sport' | 'culture' | 'lifestyle' | 'labs';
 
 type Edition = 'UK' | 'US' | 'INT' | 'AU';
+type EditionLower = 'uk' | 'us' | 'int' | 'au'; // Is there a way to do this in typescript without duplication?
```

Comment:
Can we always use Edition and then just lower case it at the locations we need a lower case value?

Reply:
I'd like to but I tried to type the use point as `Edition` but then `uk` === `UK` so not sure how to do that? https://github.com/guardian/dotcom-rendering/pull/682/files#diff-e91b0f7e5d1afa711d9cb29172bcd2feR223